appointment/appointment.go

```diff
@@ -25,21 +25,22 @@ const (
 // Appointment will contain all essential details about a appointment
 type Appointment struct {
 	Duration  time.Duration        `json:"duration" dynamodbav:"duration"`
-	Counselor *counselor.Counselor `json:"counselor" dynamodbav:"counselor"`
-	Patient   *user.User           `json:"patient" dynamodbav:"patient"`
-	Time      time.Time            `json:"time" dynamodbav:"appointmentTime"`
-	Status    int                  `json:"status" dynamodbav:"status"`
+	Counselor string `json:"counselor" dynamodbav:"counselor"`
```

Comment:
If you are using vscode, can you install go extension... this code won’t compile on other machines who use vscode with go due to formatting and styling issues

Reply:
https://github.com/golang/vscode-go

Reply:
i used it dont get any errors

Reply:
Indentation seem messed up